nixos/modules/services/x11/desktop-managers/gnome3.nix

```
@@ -126,6 +126,7 @@ in {
     services.dleyna-renderer.enable = mkDefault true;
     services.dleyna-server.enable = mkDefault true;
     services.gnome3.at-spi2-core.enable = true;
+    services.gnome3.evince.enable = true;
```

Some people might want lighter sessions without Evince & gstreamer.

```suggestion
    services.gnome3.evince.enable = mkDefault true;
```

---

Done.